Sources/SwiftFormatPrettyPrint/Token.swift

```diff
@@ -185,8 +185,8 @@ enum Token {
   case commaDelimitedRegionStart
 
   /// Marks the end of a comma delimited collection, where a trailing comma should be inserted
-  /// if and only if the collection spans multiple lines.
-  case commaDelimitedRegionEnd(hasTrailingComma: Bool)
+  /// if and only if the collection spans multiple lines, and the collection had multiple elements.
```

Comment:
`if and only if the collection spans multiple lines and has multiple elements.`

Reply:
Thanks. Updated the text.